packit_service/service/api/webhooks.py

```diff
@@ -96,7 +96,11 @@ def post(self):
 
         celery_app.send_task(
             name=getenv("CELERY_MAIN_TASK_NAME") or CELERY_DEFAULT_MAIN_TASK_NAME,
-            kwargs={"event": msg},
+            kwargs={
+                "event": msg,
+                "service": "github",
```

Comment:
When you mentioned `service`, I thought we're talking about something like `packit`, `stream`, `fedora-source-git`… Maybe I would reconsider and use something like `source`?